.storybook/preview.tsx

```diff
@@ -57,15 +57,11 @@ const preview: Preview = {
         },
       },
     },
```

Comment:
It's cuz I was handling it through Storybook. Because they have specific devices available. But I see from your comments further in the review that you're not a fan of that. I can also bring this back and remove those stories.